lib/src/syn/v1/stmt/remove.rs

```diff
@@ -178,10 +178,13 @@ pub fn table(i: &str) -> IResult<&str, RemoveTableStatement> {
 	let (i, _) = tag_no_case("TABLE")(i)?;
 	let (i, _) = shouldbespace(i)?;
 	let (i, name) = cut(ident)(i)?;
+	let (i, if_exists) =
+		opt(tuple((shouldbespace, tag_no_case("IF"), shouldbespace, tag_no_case("EXISTS"))))(i)?;
```

Comment:
You should probably cut here since after `IF` the `EXISTS` keyword should always be next.
```suggestion
		opt(tuple((shouldbespace, tag_no_case("IF"), shouldbespace, cut(tag_no_case("EXISTS")))))(i)?;
```

Reply:
Should I add a test case for that as well?

Reply:
Done. I added a new test and also cut the space before `EXISTS` for it to work.